CONTRIBUTING.md

```diff
@@ -80,6 +80,30 @@ Testing is done with [Jest](https://jestjs.io/). To run the tests:
 npm run test:jest
 ```
 
+### Environments
+
+You can manage different environments by creating JSON files and setting the `NODE_ENV` variable accordingly.
+The default environment is `dev`, which uses the file `environments/environment.dev.json`.
+
+1. Create a JSON file under the folder `environments` with the name `environment.<environment name>.json`,
+   for example: `environment.prod.json`
+2. Add your variables to the JSON object within the file.
```

Comment:
what variables are those? I guess it's some kind of app keys to insights like this ? `"key": '"value"`

Reply:
Yes, a simple JSON with whatever you need there. 
I took the example from Angular apps that have `production: true / false`.
For App Insights I added `insightsKey`, but it will work also if you don't have it in the JSON.
This way developers don't have to worry about it.